Slate UI remodified

danra2 opened the pull request:
Reverting the SlateUI changes made to selectively only update SlateUI oriented files.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/SAP-docs/preview.developer.concur.com?pullRequest=8) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/SAP-docs/preview.developer.concur.com?pullRequest=8) before we can accept your contribution.<br/><hr/>**d.ra** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SAP-docs/preview.developer.concur.com?pullRequest=8) it.</sub>

danra2 commented:
Reverted previous changes, went through the changes manually starting from Jan 29th. Adjusted all slate-ui related files, for reason the URLs weren't redirecting, my guess is that since I didn't have the Slate-Ui folder it wasn't converting the filers from the original swagger base, now that I have the folder it is redirecting properly to the files generated. Still seems kind of fishy, want to investigate further PRing for transparency.

danra2 commented:
Discarded, updated PR:
https://github.com/SAP-docs/preview.developer.concur.com/pull/11
